doc/isomorph.xml

```diff
@@ -54,6 +54,9 @@
     the value of either <C>BlissAutomorphismGroup</C> or  <Ref
       Attr="NautyAutomorphismGroup"/>. These groups are, of course, equal but
     their generating sets may differ. <P/>
+    The attribute <Ref Attr="AutomorphismGroup" Label="for a digraph,
```

Comment:
You can't have line-breaks in the label bit of a `Ref` tag - GAP doc gives warnings.